klayout/drc/rule_decks/hres.drc

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+if FEOL
+  #================================================
+  #----------------H POLY RESISTOR-----------------
+  #================================================
+
+  logger.info('Starting HRES derivations')
+  hres_poly = poly2.interacting(pplus).interacting(sab).interacting(res_mk).interacting(resistor)
+  hres1_poly = poly2.interacting(pplus).interacting(sab).interacting(res_mk)
+
+  # Rule HRES.1: Minimum space. Note : Merge if the spacing is less than 0.4 um. is 0.4µm
```

Comment:
not sure why need to repeat the constraint twice?

Reply:
Same as above